### PR TITLE
fix some importcsv tool bug

### PR DESCRIPTION
**Pull request by mychaow**

### CODE REVIEW

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java [hunk 1]
```diff
@@ -307,7 +307,7 @@ private static boolean queryDatabaseMeta(String[] strHeadInfo, File file, Buffer
         ResultSet resultSet = statement.getResultSet();
         try {
           if (resultSet.next()) {
```
Comment (jixuan1989) [inline cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:309]: can you add a comment to describe the structure of ResultSet?

**Review by jixuan1989 — COMMENTED** [review]
Better to describe what this PR is for.

And, better to add IT to cover the case.

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java (hunk 1 above)
Comment (mychaow) [reply cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:309]: ok, I will add some description. Thank you for pointing this out.

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java [hunk 2]
```diff
@@ -378,7 +388,12 @@ private static String createOneSql(Map.Entry<String, ArrayList<Integer>> entry,
         continue;
       }
       if (timeseriesToType.get(headInfo.get(colIndex.get(j))).equals(STRING_DATA_TYPE)) {
-        sbd.append(", \'").append(data[colIndex.get(j) + 1]).append("\'");
+        if ((data[colIndex.get(j) + 1].startsWith("'") && data[colIndex.get(j) + 1].endsWith("'")) ||
```
Comment (qiaojialin) [inline cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:391]: could you add some comment...  give some example

File: cross-tests/src/test/java/org/apache/iotdb/cross/tests/tools/importCsv/AbstractScript.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import static org.junit.Assert.assertTrue;
+
+public abstract class AbstractScript {
+
+  protected void testOutput(ProcessBuilder builder, String[] output) throws IOException {
+    builder.redirectErrorStream(true);
+    Process p = builder.start();
+    BufferedReader r = new BufferedReader(new InputStreamReader(p.getInputStream()));
+    String line;
+    List<String> outputList = new ArrayList<>();
```
Comment (qiaojialin) [inline cross-tests/src/test/java/org/apache/iotdb/cross/tests/tools/importCsv/AbstractScript.java:36]:
```suggestion
    List<String> actualOutput = new ArrayList<>();
```

File: cross-tests/src/test/java/org/apache/iotdb/cross/tests/tools/importCsv/AbstractScript.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      } else {
+        outputList.add(line);
+      }
+    }
+    r.close();
+    p.destroy();
+
+    for (int i = 0; i < output.length; i++) {
+      System.out.println("output:" + output[i]);
+      System.out.println("outputList:" + outputList.get(i));
```
Comment (qiaojialin) [inline cross-tests/src/test/java/org/apache/iotdb/cross/tests/tools/importCsv/AbstractScript.java:50]:
```suggestion
      System.out.println("expect:" + output[i]);
      System.out.println("actual:" + outputList.get(i));
```


Or remove the print

Comment (mychaow) [reply cross-tests/src/test/java/org/apache/iotdb/cross/tests/tools/importCsv/AbstractScript.java:50]: ok

File: cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java (hunk 2 above)
Comment (mychaow) [reply cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java:391]: ok

**Review by qiaojialin — COMMENTED** [review]
Hi, One more issue is to add the cross-tests module in travis.yml build command

**Review by qiaojialin — APPROVED** [review]

File: cross-tests/src/test/java/org/apache/iotdb/cross/tests/tools/importCsv/AbstractScript.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        break;
+      } else {
+        actualOutput.add(line);
+      }
+    }
+    r.close();
+    p.destroy();
+
+    for (int i = 0; i < output.length; i++) {
+      assertTrue(actualOutput.get(i).startsWith(output[i]));
```
Comment (samperson1997) [inline cross-tests/src/test/java/org/apache/iotdb/cross/tests/tools/importCsv/AbstractScript.java:49]: Hi, the `ImportCsvTestIT` keeps failing occasionally in windows environment of Travis these days. @Genius-pig could also reproduce this failure in his local PC. Could you please have another check? I created [an issue](https://issues.apache.org/jira/browse/IOTDB-935) in JIRA : )

**Review by samperson1997 — COMMENTED** [review]
A laaaaaate review ....

**Pull request merged**